spec/system/composer/dont_feed_the_trolls_popup_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+describe "Composer don't feed the trolls popup", type: :system, js: true do
+  fab!(:user) { Fabricate(:user) }
+  fab!(:troll) { Fabricate(:user) }
+  fab!(:topic) do
+    Fabricate(:topic, user: user, category: Category.find(SiteSetting.uncategorized_category_id))
+  end
```

Comment:
```suggestion
  fab!(:topic) { Fabricate(:topic, user: user) } 
```

I don't think we need to pass the fabricator the `uncategorized` category here since that is what it defaults to:

https://github.com/discourse/discourse/blob/cb1e95cd3d16c1550596cee33ef361402d1bec0b/spec/fabricators/topic_fabricator.rb